v8/payloads/channel.ts

```diff
@@ -99,6 +99,12 @@ export interface APIChannel extends APIPartialChannel {
 	 * This may be `null` in events such as `GUILD_CREATE` when a message is not pinned
 	 */
 	last_pin_timestamp?: string | null;
+	/**
+	 * The camera video quality mode of the voice channel, 1 when not present
```

Comment:
Although we should probably not document the raw value, but the enum ref instead
```suggestion
	 * The camera video quality mode of the voice channel, `1` when not present
```